scripts/variantstore/wdl/GvsPopulateAltAllele.wdl

```diff
@@ -232,6 +250,6 @@ task PopulateAltAlleleTable {
   }
 
   output {
-    String done = "~{vet_table_name}"
+    String done = "~{sep=' ' vet_table_names}"
```

Comment:
I've been using `done` as a boolean to indicate when the call is done so tasks dependent on side effects will know when to run. It seems this is actually a different concept, whitespace separated vet table names.